src/RoutingControllersOptions.ts

```diff
@@ -20,6 +20,11 @@ export interface RoutingControllersOptions {
     routePrefix?: string;
 
     /**
+     * Enable automatic fallthrough from action handlers to future actions or middleware
+     */
+    automaticFallthrough?: boolean;
```

Comment:
> Also TBH fallthrough does not indicate for me what this option does, but I can't find a good name for it. Maybe this terminology is highly used, simply Im not familiar with it

It should be `FallThrough` because there is no such word as _fallthrough_, it is _fall through_. It is used in programming so I am cool with it, however we can make it more explicit via naming it as `routeFallThrough`.